src/stories/badge.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Badge as ChakraBadge, BadgeProps, Container } from "@chakra-ui/react"
+import { Meta, Story } from "@storybook/react"
+import { badgeVariants, colorSchemes, sizes } from "../../.storybook/chakraVars"
+
+const Template: Story<BadgeProps> = ({ ...args }) => {
+  return (
+    <Container>
+      <ChakraBadge {...args} fontSize={args.fontSize}>
+        Badge
+      </ChakraBadge>
+    </Container>
+  )
+}
+
+export const Badge = Template.bind({
+  colorScheme: "brand",
+  variant: "solid",
+  size: "sm",
+})
+
+export default {
+  title: "Badge",
+  component: Badge,
+  argTypes: {
+    colorScheme: {
+      options: colorSchemes,
+      control: { type: "radio" },
+    },
```

Comment:
As Rafał already mentioned we should document more our stories. Here i would add some description and default values which are specified in `Badge.ts`:

```suggestion
  argTypes: {
    colorScheme: {
      options: colorSchemes,
      description: "Changes the color of the badge"
      control: { type: "radio" },
      table: {
        defaultValue: {summary: 'brand'} //this is for setting default value
      },
```

We should add similar properties to all argTypes both in `Badge` and `Button`